plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -311,6 +311,17 @@ def template_for_update
     end
 
     class Service < VersionedUpdate
+      def update
+        super
+      rescue Samson::Hooks::UserError => e
+        if e.message.include?("cannot change") # original error got already translated in #request
```

Comment:
make it more precise: `cannot change healthCheckNodePort`

Reply:
it's meant as a catch-all since there are more things that will cause the "cannot change"

Reply:
we may have "can't change" because of other issues.
I don't like the idea to catch all in this case.

Reply:
it's opt in ... so user has to decide if they want to do force update or not ... but this way we don't have to hardocde all the millions of things that are not updateable